open_event/helpers/helpers.py

```diff
@@ -31,14 +31,25 @@ def is_track_name_unique_in_event(form, event_id, *args):
                       "VwmIvKlPoc1YVpKCSOwhEFWZvxFT8")
 }
 
+def send_email_invitation(email, username, event_name, link):
+    """Send email after account create"""
+    payload = {'to': email,
+               'from': 'open-event@googlegroups.com',
+               'subject': "Invitation to Submit Papers for " + event_name,
+               "html": ("Hi %s<br/>" + \
+                        "You are invited to submit papers for event: %s" + \
+                        "<br/> Visit this link to fill up details: %s" % (username, event_name, link))}
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Too many arguments for format string](https://www.codacy.com/app/triplez/open-event-orga-server/file/2674855889/issues/source?bid=3233864&fileBranchId=3324174#l41)